tokio/src/sync/mpsc/chan.rs

```diff
@@ -25,6 +26,16 @@ impl<T, S: fmt::Debug> fmt::Debug for Tx<T, S> {
     }
 }
 
+pub(crate) struct TxWeak<T, S> {
+    inner: Weak<Chan<T, S>>,
+}
```

Comment:
Loom doesn't support `Weak`. Regardless, an ordinary `Arc` should still work I think.